dio/lib/src/sentry_transformer.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:dio/dio.dart';
+import 'package:sentry/sentry.dart';
+
+/// A transformer which wraps transforming in spans
+class SentryTransformer implements Transformer {
+  // ignore: public_member_api_docs
+  SentryTransformer({required Transformer transformer, Hub? hub})
+      : _hub = hub ?? HubAdapter(),
+        _transformer = transformer;
+
+  final Transformer _transformer;
+  final Hub _hub;
+
+  @override
+  Future<String> transformRequest(RequestOptions options) async {
+    final span = _hub.getSpan()?.startChild(
+          'serialize',
+          description: 'Dio.transformRequest: ${options.method} ${options.uri}',
```

Comment:
`serialize` already follows the convention https://develop.sentry.dev/sdk/performance/span-operations/#web-server
I'd rather remove the `Dio.transformRequest` prefix from the description, wdyt?

Reply:
I'm always quite unsure what to use as operation and description. So if you're saying dropping the prefix makes sense it's fine for me. I've included it, so that you're able to see where it comes from.

Reply:
both are raw Strings and there's a convention, but its not enforced anywhere, some UI features rely on the span operation, that's why the docs page, description is open, yet.
I'd remove the `Dio.transformRequest:` since the method/ URL and span operation already tells you where it comes from